rewrite-gradle/src/main/java/org/openrewrite/gradle/search/FindDependency.java

```diff
@@ -62,18 +64,38 @@ public String getInstanceNameSuffix() {
 
     @Override
     public String getDescription() {
-        return "Finds dependencies declared in `build.gradle` files. See the [reference](https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_configurations_graph) on Gradle configurations or the diagram below for a description of what configuration to use. " +
+        return "Finds dependencies declared in gradle build files. See the [reference](https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_configurations_graph) on Gradle configurations or the diagram below for a description of what configuration to use. " +
                 "A project's compile and runtime classpath is based on these configurations.\n\n<img alt=\"Gradle compile classpath\" src=\"https://docs.gradle.org/current/userguide/img/java-library-ignore-deprecated-main.png\" width=\"200px\"/>\n" +
                 "A project's test classpath is based on these configurations.\n\n<img alt=\"Gradle test classpath\" src=\"https://docs.gradle.org/current/userguide/img/java-library-ignore-deprecated-test.png\" width=\"200px\"/>.";
     }
 
+    private static final List<String> DEPENDENCY_MANAGEMENT_METHODS = Arrays.asList(
```

Comment:
Maybe it would make more sense to put this in a global file?

Reply:
This should probably transition to the `GradleDependency` trait which doesn't need the list at all.